sdks/java/io/pom.xml

```diff
@@ -59,6 +59,7 @@
     <module>mqtt</module>
     <module>redis</module>
     <module>solr</module>
+    <module>text</module>
```

Comment:
<!--new_thread; commit:c6c7070ad92424707d3720d3a4dc2c0fb6961440; resolved:0-->
Can we rename this to "file-based-io-tests" (looks like there we already have elasticsearch-tests, so there's a precedence) ?

Reply:
ok, I like this idea due to two reasons: 
- we'll have a common pom.xml and all test dependencies written once for all file based IO ITs and this is nice because it seems all text IOITs will have the same dependencies, need the same profiles, etc. 
- we can move TestTextLine to this module and use it only there (because nowhere else we should use it but in file based IOITs).